log_file.log

```diff
@@ -0,0 +1,12 @@
+2019-04-10 17:58:08 INFO  ConnectorUtil:28 - info logger works correctly
+2019-04-11 13:40:09 ERROR ConnectorUtil:37 - Errorjava.nio.file.NoSuchFileException: database.properties
```

Comment:
Does your code work?
I don't think it is a good idea to push log file into a repository. Please add it into gitignore.

Reply:
Everything is work correctly